include/ur.py

```diff
@@ -1779,7 +1779,6 @@ class ur_function_v(IntEnum):
     PLATFORM_GET_API_VERSION = 74                   ## Enumerator for ::urPlatformGetApiVersion
     PLATFORM_GET_NATIVE_HANDLE = 75                 ## Enumerator for ::urPlatformGetNativeHandle
     PLATFORM_CREATE_WITH_NATIVE_HANDLE = 76         ## Enumerator for ::urPlatformCreateWithNativeHandle
-    GET_LAST_RESULT = 77                            ## Enumerator for ::urGetLastResult
```

Comment:
Do we want to add a comment calling out `77` as available for reuse?

Or add a comment saying `77 was previously GET_LAST_RESULT`?

Reply:
> Or add a comment saying `77 was previously GET_LAST_RESULT`?

I'm not sure how that would work in `scipts/core/registry.yml`.

Reply:
Okay, if it's automatically generated not worth the effort to change the generator.